code/src/App.js

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+/* eslint-disable no-trailing-spaces */
+import React, { useState, useEffect } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import MovieList from 'components/MovieList';
+// import NotFound from 'components/NotFound';
+// We never used this but we keep it for future usage if necessary
+import MovieDetails from 'components/MovieDetails';
+import { BASE_URL } from 'utils/urls';
 
 export const App = () => {
+  const [movieList, setMovieList] = useState([]);
+  
+  useEffect(() => {
+    fetch(BASE_URL)
+      .then((data) => data.json())
+      .then((configuredData) => setMovieList(configuredData.results))
```

Comment:
You mentioned this in the Demo. Maybe because you already used the word 'data' in the .then before, so it just had to be different word here?